omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -278,6 +278,19 @@ public SimpleObject submit(UiSessionContext sessionContext, @RequestParam("perso
 				    new FormSubmissionError("general-form-error", ui.message("htmlformentryui.datetimeBeforeVisitDate")));
 			}
 			
+			if (validationErrors.size() > 0) {
+				return returnHelper(validationErrors, fes, null);
+			}
+		} else if(visit != null) {
```

Comment:
thanks @icrc-toliveira we can put this on the same line

Reply:
Hi @HerbertYiga 
I´m not sure if this was you mean, but I change the indentation to be on the same line.
However, I noticed that if I let the project auto-ident, the entire file will have changes